R/router.R

```diff
@@ -1,18 +1,37 @@
 ROUTER_UI_ID <- '_router_ui'
 
+#' Create a mapping bewtween a ui element
+#' and a server callack
+#'
+#' @param ui Valid Shiny user interface.
+#' @param server Function that is called within the global server function if given
+callback_mapping <- function(ui, server = NA) {
+  server <- if (is.function(server)) {
```

Comment:
what about `ifelse` here?

Reply:
I think ifelse should be used to perform vectorisation. `callback_mapping` is not dedicated to it I suppose, see 16 and 17 line. Is there any other reason to do it?

Reply:
Yes, agree. I prefer to use it sometimes for neater styling though. In that case NVM.

Reply:
👍 I have no strong opinion on that, both variants are arguable to me. Let's roll with the one you feel most comfortable with!